ground/gcs/src/plugins/sysalarmsmessaging/sysalarmsmessagingplugin.cpp

```diff
@@ -89,8 +105,6 @@ bool SysAlarmsMessagingPlugin::initialize(const QStringList &arguments, QString
  */
 void SysAlarmsMessagingPlugin::updateAlarms(UAVObject* systemAlarm)
 {
-    Q_ASSERT(systemAlarm->getObjID() == SystemAlarms::OBJID);
```

Comment:
Good question. I made the change

```
   // Check that the UAVO library is in sync with the plugin. If not, return.
if (systemAlarm->getObjID() != SystemAlarms::OBJID) {
    return;
}
```

but I'm not actually sure if we should specifically handle that case here. The function will terminate normally without having done anything, which is what we expect of it. The problem of having UAVOs out of sync with plugins is universal, and we don't test in other plugins so I'm inclined to think this test would be out of place.

Still, the code is here and ready to go. I can merge it in at reviewer request.